pkg/cmd/server/start/start_master.go

```diff
@@ -474,16 +474,15 @@ func StartAPI(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) error {
 	}
 
 	// verify we can connect to etcd with the provided config
-	// TODO(post-1.6.1-rebase): uncomment etcd3 support
```

Comment:
we decided to postpone etcd3 to post-rebase. Andy started with it, but ran into issues.